repositories.yaml

```diff
@@ -205,6 +203,7 @@ applications:
       - browser/components/search/metrics.yaml
       - browser/components/urlbar/metrics.yaml
       - browser/modules/metrics.yaml
+      - toolkit/components/crashes/metrics.yaml
```

Comment:
just above:
```
metrics_files: # When adding here, consider if you should also add to pine.
```

do we need to add these changes to pine?

Reply:
We do not. `pine` is about a project that was cancelled some months ago. If you'd like I can file a follow-up to remove the comment.